paper-drawer-panel.html

```diff
@@ -506,13 +516,22 @@
           tap: '_onTap',
           track: '_onTrack',
           down: '_downHandler',
-          up: '_upHandler'
+          up: '_upHandler',
+          transitionend: '_onMainTransitionEnd'
```

Comment:
Should we change this to `_onTransitionEnd` since we are not listening just on the `#main` anymore?